cardano_clusterlib/clusterlib.py

```diff
@@ -1692,6 +1692,46 @@ def get_kes_period(self) -> int:
         """Return last block KES period."""
         return int(self.get_slot_no() // self.slots_per_kes_period)
 
+    def get_kes_period_info(
+        self, issue_op_cert: FileType, operational_certificate_issue_counter: FileType
```

Comment:
We already use name `opcert_file` for your `issue_op_cert`, and `cold_counter_file` for your `operational_certificate_issue_counter` elsewhere in tests and/or clusterlib.